plugins/@grouparoo/spec-helper/src/lib/factories/profile.ts

```diff
@@ -9,7 +10,21 @@ const data = async (props = {}) => {
   return Object.assign({}, defaultProps, props);
 };
 
-export default async (props = {}) => {
+export default async (props = {}, properties = {}) => {
   const { Profile } = await import(`@grouparoo/core/${loadPath}`);
-  return Profile.create(await data(props));
+  const profile = await Profile.create(await data(props));
+
+  const { Property } = await import(`@grouparoo/core/${loadPath}`);
+  const allProperties = await Property.findAllWithCache();
+  const directlyMappedProperty = allProperties.find((p) => p.directlyMapped);
+
+  if (directlyMappedProperty) {
+    properties[directlyMappedProperty.key] = faker.datatype.number(99999);
+  }
```

Comment:
Think there could be collisions in tests because of this.  Does faker return a random number or something smarter?

Reply:
Changed to use `faker.unique` to ensure uniqueness!